bciers/apps/registration1/e2e/poms/operations.ts

```diff
@@ -313,7 +313,6 @@ export class OperationsPOM {
               this.buttonConfirmModal,
               this.alertDeclined,
             );
-            await expect(this.messageOperationDeclined).toBeVisible();
             break;
```

Comment:
Removed this as I believe it was erroneously checking for a declined messaged that appeared due to a bug:

This is what appears in `develop` branch currently as the revalidate tag causes rerender:
<img width="1402" alt="Screenshot 2024-06-18 at 3 48 47 PM" src="https://github.com/bcgov/cas-registration/assets/14259474/6bd73b0f-927c-4acb-a96a-86b6e7edb7ee">

This is what appears with the revalidate url removed:
<img width="1402" alt="Screenshot 2024-06-18 at 3 49 51 PM" src="https://github.com/bcgov/cas-registration/assets/14259474/b322767d-62d1-4dd3-bb70-00c0c2ecc4a2">

Note: the alert message immediately disappears with this updated version of nextjs in this PR without removing the revalidate tag which is why e2e was failing.

Reply:
> Removed this as I believe it was erroneously checking for a declined messaged that appeared due to a rerendering bug. Both alert messages show at the same time, when only the initial alert should show up unless you refresh the page as the other alert is permenant.
> 
> This is what appears in `develop` branch currently as the revalidate tag causes rerender: <img alt="Screenshot 2024-06-18 at 3 48 47 PM" width="1402" src="https://private-user-images.githubusercontent.com/14259474/340860392-6bd73b0f-927c-4acb-a96a-86b6e7edb7ee.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3NTE4NDUsIm5iZiI6MTcxODc1MTU0NSwicGF0aCI6Ii8xNDI1OTQ3NC8zNDA4NjAzOTItNmJkNzNiMGYtOTI3Yy00YWNiLWE5NmEtODZiNmU3ZWRiN2VlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDIyNTkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4NzQ3MjA5ZWEyYzY1ZGY4MjgxYmY0MmNmYTJkOTkyZDJiNDU0OWFlYjgzYTBiN2ZjOWU3ZDA0MDA2NjBmOTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._rljlWydetYo4sb14rPAdM-r5sWPEAqnnMmayfQ5FU4">
> 
> This is what appears in this branch with updated nextjs/react and the revalidate url removed. If we don't fix the revalidate tag this message will immediately disappear in this PR due to rerendering which is why e2e was failing. <img alt="Screenshot 2024-06-18 at 3 49 51 PM" width="1402" src="https://private-user-images.githubusercontent.com/14259474/340860549-b322767d-62d1-4dd3-bb70-00c0c2ecc4a2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3NTE4NDUsIm5iZiI6MTcxODc1MTU0NSwicGF0aCI6Ii8xNDI1OTQ3NC8zNDA4NjA1NDktYjMyMjc2N2QtNjJkMS00ZGQzLWJiNzAtMDBjMGMyZWNjNGEyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDIyNTkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0OTZhY2Q1MTFkNTdlZmE5NGM3MGMwNmM2ZDAyZjg1OTYyNzBkNTIzNTAzZDc2NjVmYzZlODBmNTdjMzQxMGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vTq1HpczHsIgLVtsUmfjictMZRz6x4BZn7CFFdzrOIc">



> Removed this as I believe it was erroneously checking for a declined messaged that appeared due to a rerendering bug. Both alert messages show at the same time, when only the initial alert should show up unless you refresh the page as the other alert is permenant.
> 
> This is what appears in `develop` branch currently as the revalidate tag causes rerender: <img alt="Screenshot 2024-06-18 at 3 48 47 PM" width="1402" src="https://private-user-images.githubusercontent.com/14259474/340860392-6bd73b0f-927c-4acb-a96a-86b6e7edb7ee.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3NTE4NDUsIm5iZiI6MTcxODc1MTU0NSwicGF0aCI6Ii8xNDI1OTQ3NC8zNDA4NjAzOTItNmJkNzNiMGYtOTI3Yy00YWNiLWE5NmEtODZiNmU3ZWRiN2VlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDIyNTkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4NzQ3MjA5ZWEyYzY1ZGY4MjgxYmY0MmNmYTJkOTkyZDJiNDU0OWFlYjgzYTBiN2ZjOWU3ZDA0MDA2NjBmOTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._rljlWydetYo4sb14rPAdM-r5sWPEAqnnMmayfQ5FU4">
> 
> This is what appears in this branch with updated nextjs/react and the revalidate url removed. If we don't fix the revalidate tag this message will immediately disappear in this PR due to rerendering which is why e2e was failing. <img alt="Screenshot 2024-06-18 at 3 49 51 PM" width="1402" src="https://private-user-images.githubusercontent.com/14259474/340860549-b322767d-62d1-4dd3-bb70-00c0c2ecc4a2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3NTE4NDUsIm5iZiI6MTcxODc1MTU0NSwicGF0aCI6Ii8xNDI1OTQ3NC8zNDA4NjA1NDktYjMyMjc2N2QtNjJkMS00ZGQzLWJiNzAtMDBjMGMyZWNjNGEyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDIyNTkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0OTZhY2Q1MTFkNTdlZmE5NGM3MGMwNmM2ZDAyZjg1OTYyNzBkNTIzNTAzZDc2NjVmYzZlODBmNTdjMzQxMGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vTq1HpczHsIgLVtsUmfjictMZRz6x4BZn7CFFdzrOIc">


To test this check out this branch and:

- run `/bciers && yarn && nx reset` 
-  log in as internal user and go to the Operations grid and click `View details` of a `Pending` Operation
- Approve or Confirm the Operations. You should only see the green personalized alert. If you refresh the page you will see the generic permenant alert.